src/components/Timer/timer.js

```diff
@@ -94,6 +95,19 @@ export default class Timer extends Component {
             <span className="material-symbols-outlined">replay</span>
           </button>
         </div>
+        <div>
+          <div style={{ width: '100%', border: '1px solid #ccc', marginTop: '10px' }}>
+            <div
+              style={{
+                width: `${(this.state.pomodoroCount % 4) * 25}%`,
+                height: '20px',
+                backgroundColor: 'green',
+                transition: 'width 0.5s ease-in-out',
+              }}
+            />
+          </div>
+          <p>{`${this.state.pomodoroCount % 4}/4 pomodoros until large break!`}</p>
```

Comment:
I would change what this line says, but for now is okay, I wasn't able to think of another one.